packages/backend/src/graphql/mutations/update-flow-config.ts

```diff
@@ -0,0 +1,34 @@
+import { type NotificationFrequency } from '@/models/flow'
+import Context from '@/types/express/context'
+
+type Params = {
+  input: {
+    id: string
+    notificationFrequency: NotificationFrequency
+  }
+}
+
+const updateFlowConfig = async (
+  _parent: unknown,
+  params: Params,
+  context: Context,
+) => {
+  let flow = await context.currentUser
+    .$relatedQuery('flows')
+    .findOne({
+      id: params.input.id,
+    })
+    .throwIfNotFound()
+
+  flow = await flow.$query().patchAndFetch({
+    config: {
+      errorConfig: {
+        notificationFrequency: params.input.notificationFrequency,
+      },
+    },
+  })
```

Comment:
Objection's `patch` is not JSON aware so this will wipe out other parts of the config.

Something like this will fix the issue (need to add checks for null, etc though):
```suggestion
    config: {
      ...flow.config,
      errorConfig: {
        ...flow.config.errorConfig,
        notificationFrequency: params.input.notificationFrequency,
      },
    },
  })
```

Reply:
thanks for the catch!